### PR TITLE
8.0 move sale payment method

### PR DESCRIPTION
**Pull request by bguillot**
This PR moves the module sale_payment_method from e-commerce repo.

See https://github.com/OCA/e-commerce/issues/47 for the reason and discussions about that.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hello, thank you for this work. Could you please check travis? I think we can go on with this move.

**Comment by lepistone** [comment]
Hi! I :+1: .

The failure on one of the travis builds seems unrelated and intermittent (I restarted the job). However, this needs a companion PR to remove the module from `e-commerce`. I'll take care of this. In the meantime can someone please approve this?

**Comment by lepistone** [comment]
The build is green. Two minor commits that were done after this PR are missing, but they are just an icon and prefixing the version with `8.0`. I will add those in a subsequent PR. I think we can merge this.

Thanks!

**Comment by gurneyalex** [comment]
:+1:

**Comment by mdietrichc2c** [comment]
:+1:

File: sale_payment_method/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{'name': 'Sale Payment Method',
+ 'version': '0.2.1',
```
Comment (tafaRU) [inline sale_payment_method/__openerp__.py:23]: Please update it according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: sale_payment_method/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ 'author': "Akretion,Odoo Community Association (OCA)",
+ 'website': 'http://www.akretion.com/',
+ 'depends': ['sale',
+             ],
+ 'data': ['sale_view.xml',
+          'payment_method_view.xml',
+          'security/ir.model.access.csv',
+          'security/rules.xml',
+          ],
+ 'demo': [],
```
Comment (tafaRU) [inline sale_payment_method/__openerp__.py:35]: Please remove empty key.

File: sale_payment_method/README.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
+Sale Payment Method
```
Comment (tafaRU) [inline sale_payment_method/README.rst:1]: Please add above another row of equal signs, see https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst as reference.

**Comment by tafaRU** [comment]
My minor remarks can be addressed in another PR as written at https://github.com/OCA/sale-workflow/pull/172#issuecomment-160924864

I merge then.

**Comment by IgnacioMelendrez** [comment]
Hi, I am installing WooCommerce in Odoo 8. Now I have the WooCommerce in the application list, but when I install I get I should install me4 get this error (try to install the module 'connector_ecommerce' depending on the module 'sale_payment_method_automatic_workflow'.
The latter module is not available on your system.) I have 'sale_payment_method_automatic_workflow', 'sale_automatic_workflow' and 'sale_payment_method' in the addons but still I get the same error. Can someone please help me. Thank you.

**Pull request merged**